src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -363,7 +363,10 @@ public static function create($uri, $method = 'GET', $parameters = array(), $coo
             case 'POST':
             case 'PUT':
             case 'DELETE':
-                if (!isset($server['CONTENT_TYPE'])) {
+                if (isset($server['content_type']) {
```

Comment:
This sound like an integration scenario.
But is always possible to pass anything you want into the $server parameter, e.g. in a functional test.